Dockerfile.8.0

```diff
@@ -0,0 +1,23 @@
+ARG SDK_VERSION_TAG
+ARG DOCKER_REPOSITORY=mcr.microsoft.com/dotnet/sdk
+FROM ${DOCKER_REPOSITORY}:${SDK_VERSION_TAG}
+
+ARG OPENJDK_PACKAGE=openjdk-11-jre
+
+LABEL maintainer="estafette.io" \
+      description="The estafette-extension-dotnet component is an Estafette extension to build and publish .NET Core applications and libraries."
+
+RUN grep security /etc/apt/sources.list | tee /etc/apt/security.sources.list && \
+    apt-get update && \
+    apt-get upgrade -y -o Dir::Etc::SourceList=/etc/apt/security.sources.list && \
+    apt-get install -yq \
+        $OPENJDK_PACKAGE && \
```

Comment:
@katesoglu 
-  You need to remove temp source APT packages from the docker image in the same step of installation. This will help to get slimmer docker images (faster builds)

```
apt-get clean && rm -rf /var/lib/apt/lists/*
```
- Also, it is preferred to run the docker image with non root user (Optional: as current extentions are running with root user :) )